cursorless-talon/src/actions/homophones.py

```diff
@@ -1,10 +1,13 @@
+from typing import Optional
+
 from talon import actions, app
 
+from ..targets.target_types import CursorlessTarget
 from .get_text import cursorless_get_text_action
 from .replace import cursorless_replace_action
 
 
-def cursorless_homophones_action(target: dict):
+def cursorless_homophones_action(target: CursorlessTarget):
```

Comment:
This was straight up wrong, given how it was called and what it calls. Noticed this thanks to Pylance's type checking 😄